docs/docs/1-getting-started/4-crud-generator/1-api-generator.md

```diff
@@ -6,6 +6,14 @@ id: api-generator
 
 The API Generator can be used to generate the usual CRUD operations in GraphQL for an entity.
 
+## Installation
+
+The API Generator is inside the `@comet/api-generator` package.
+
+```bash
+npm install --dev @comet/api-generator
+```
+
```

Comment:
- the api-generator will be installed in starter, I assume
- will it be installed in an upgrade script?

if both yes, we could remove that installation section - it's always installed

Reply:
It will be updated in the starter, and an update script will be available.

Reply:
UpdateScript: https://github.com/vivid-planet/comet-upgrade/pull/65